easybuild/easyconfigs/r/RabbitMQ/RabbitMQ-3.13.1-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,55 @@
+easyblock = 'ConfigureMake'
+
+name = 'RabbitMQ'
+version = '3.13.1'
+
+homepage = 'https://www.rabbitmq.com/'
+description = """RabbitMQ is a reliable and mature messaging and streaming broker, which is easy to deploy on cloud
+environments, on-premises, and on your local machine. It is currently used by millions worldwide."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+sources = ['v%(version)s.tar.gz']
+source_urls = ['https://github.com/rabbitmq/rabbitmq-server/archive/refs/tags/']
+
+checksums = ['62ecd9567935413283700e9ed11c1aae98846e3d9ba4d9ef68b8bbc0e96c6296']
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('libxslt', '1.1.38'),
+    ('xmlto', '0.0.28'),
+    ('Python', '3.11.3'),
+]
+dependencies = [
+    ('ErlangOTP', '26.2.4'),
+    ('Elixir', '1.16.2'),
+]
+
+local_project_version_cmd = "export PROJECT_VERSION='%(version)s' && "
+
+configure_without_installdir = True
+configure_cmd = "sed -i 's|^PREFIX *?=.*$|PREFIX ?= %(installdir)s|' Makefile"
+
+prebuildopts = local_project_version_cmd
+
+preinstallopts = local_project_version_cmd
+installopts = 'install install-bin'
+postinstallcmds = ['cd %(installdir)s && ln -s lib/erlang/bin bin']
+
+test_cmd = "make tests"
+
+sanity_check_paths = {
+    'files': [
+        'bin/rabbitmqctl', 'bin/rabbitmq-diagnostics', 'bin/rabbitmq-plugins', 'bin/rabbitmq-server',
+        'bin/rabbitmq-upgrade', 'bin/rabbitmq-defaults', 'bin/rabbitmq-env', 'bin/rabbitmq-queues',
+        'bin/rabbitmq-streams', 'bin/vmware-rabbitmq'
+    ],
+    'dirs': [
+        'lib/erlang/bin', 'lib/erlang/lib',
+    ]
+}
+
+
+moduleclass = 'tools'
+
+max_parallel = 1
```

Comment:
I'd move this all the way up above `configure_without_installdir` if it is really needed

Reply:
Done in [12fe6ff](https://github.com/easybuilders/easybuild-easyconfigs/pull/20441/commits/12fe6ff03e2d7abe4fddf83b9dd0777fefe440ea)